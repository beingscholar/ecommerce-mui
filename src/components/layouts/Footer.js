import React from 'react';
import Typography from '@material-ui/core/Typography';
import { makeStyles } from '@material-ui/core/styles';

const useStyles = makeStyles((theme) => ({
    root:{
        textAlign:"center",
        paddingTop:theme.spacing(3),
    },

}))

export default function Footer(){
    const classes=useStyles();
    return(
        <div className={classes.root}>
            <Typography variant="subtitle1">Copyright Â© Your Website 2020.</Typography>
        </div>
    )
}