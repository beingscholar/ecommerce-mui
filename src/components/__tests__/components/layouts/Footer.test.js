import React from 'react'
import {render, cleanup} from '@testing-library/react'
import Footer from '../../../layouts/Footer';

 afterEach(cleanup)
 
 it('should take a snapshot', () => {
    const { asFragment } = render(<Footer />)
    
    expect(asFragment(<Footer />)).toMatchSnapshot()
   })