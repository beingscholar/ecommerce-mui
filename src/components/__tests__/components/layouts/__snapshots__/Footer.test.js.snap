// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="makeStyles-root-1"
  >
    <h6
      class="MuiTypography-root MuiTypography-subtitle1"
    >
      Copyright Â© Your Website 2020.
    </h6>
  </div>
</DocumentFragment>
`;
